led_strip/CMakeLists.txt

```diff
@@ -4,6 +4,16 @@ if(CONFIG_SOC_RMT_SUPPORTED)
     list(APPEND srcs "src/led_strip_rmt_dev.c" "src/led_strip_rmt_encoder.c")
 endif()
 
+
+list(APPEND srcs "src/led_strip_spi_dev.c")
+
+# Add here if the component is compatible with IDF >= v5.1
+if("${IDF_VERSION_MAJOR}.${IDF_VERSION_MINOR}" VERSION_GREATER_EQUAL "5.1")
+
+    list(APPEND EXTRA_COMPONENT_DIRS ../bdc_motor ../led_strip ../sh2lib ../nghttp ../esp_serial_slave_link ../onewire_bus ../ccomp_timer)
+
+endif()
+
```

Comment:
these changes should be made to https://github.com/espressif/idf-extra-components/blob/master/test_app/CMakeLists.txt